lib/api/2.0/views.js

```diff
@@ -0,0 +1,99 @@
+// Copyright 2016, EMC Inc.
+
+'use strict';
+
+var injector = require('../../../index.js').injector;
+var controller = injector.get('Http.Services.Swagger').controller;
+var views = injector.get('Views');
+var Promise = injector.get('Promise');
+var Errors = injector.get('Errors');
+var _ = injector.get('_');
+var bodyParser = require('body-parser');
+
+/**
+* @api {get} /api/2.0/views GET /views
+* @apiVersion 2.0.0
+* @apiDescription Get list of views
+* @apiName views-get
+* @apiGroup views
+* @apiSuccess {json} List of all views or if there are none an empty collection.
+*/
+var viewsGet = controller(function() {
+    return views.getAll();
+});
+
+/**
+* @api {get} /api/2.0/views/:identifier GET /views/:identifier
+* @apiVersion 2.0.0
+* @apiDescription get a specific view
+* @apiName views-getById
+* @apiGroup views
+* @apiParam {String} name of view
+* @apiSuccess {json} view with specified identifier.
+* @apiError NotFound There is no view with specified name
+*/
+var viewsGetById = controller(function(req) {
```

Comment:
Right now, I'm only using global scope.  I'm considering using scope to link resources to the set of views that can be used to present them (e.g. all views with scope 'node' could be used to render node resources).  To do this, I'd probably add views/<scope>/ and views/<scope>/<name> and GetById would become GetScopeByName or something along those lines.